src/main/play-doc/general/ReleaseNotes.md

```diff
@@ -18,6 +19,22 @@ The new licensing mechanism permits a single ConductR Agent to be used absolutel
 
 Please visit the [migration guide](MigrationGuide#Production_Suite_Licensing) for additional instructions.
 
+### External Service Registration
+
+Since 2.0, ConductR has supported a DNS SRV record fallback mechanism for service discovery i.e. if a service cannot be discovered as being hosted by ConductR, then DNS SRV lookups can be performed in order to resolve the service's location. This was disabled for standalone mode, but enabled for DC/OS installation so that Elasticsearch lookups can utilize the DC/OS hosted service.
+
+As of 2.1, ConductR replaces the 2.0 mechanism with something considerably more flexible. Any service name may be registered via ConductR's configuration to exist as an external service given one of the following mappings:
+
+* an explicit IP and port; 
```

Comment:
Semicolons, fullstops and or's should not be used in bullet points. Also, the first letter should be upper case. What about:

* IP address and port
* DNS address and port
* DNS SRV host